j/java-common-protos/java-common-protos-ubuntu_18.04.sh

```diff
@@ -0,0 +1,100 @@
+# ----------------------------------------------------------------------------
+#
+# Package       : java-common-protos
+# Version       : 2.3.2
+# Source repo   : https://github.com/googleapis/java-common-protos
+# Tested on     : Ubuntu_18.04
+# Passing Arguments: 1.Version of package, 2.JDK version (openjdk-8-jdk or openjdk-11-jdk)
+# Script License   : Apache License, Version 2 or later
+# Maintainer    : Kishor Kunal Raj <kishore.kunal.mr@ibm.com> / Priya Seth<sethp@us.ibm.com>
+#
+# Disclaimer: This script has been tested in root mode on given
+# ==========  platform using the mentioned version of the package.
+#             It may not work as expected with newer versions of the
+#             package and/or distribution. In such case, please
+#             contact "Maintainer" of this script.
+#
+# ----------------------------------------------------------------------------
+#/bin/bash
+
+export REPO=https://github.com/googleapis/java-common-protos
+
+#Default tag v2.3.2
+if [ -z "$1" ]; then
+  export VERSION="v2.3.2"
+else
+  export VERSION="$1"
+fi
+
+
+#Default testing on jdk8
+if [ -z "$2" ]; then
+  export JDK="openjdk-8-jdk"
+else
+  export JDK="$2"
+fi
+
+
+#Default installation
+sudo apt-get update
+sudo apt-get install -y apt-utils
+sudo apt-get install  git -y
+
+
+#From rerunning build
+if [ -d "flowable-engine" ] ; then
```

Comment:
how does this relate to flowable-engine?

Reply:
> how does this relate to flowable-engine?

Hi @seth-priya, I had fixed the error. Please verify